log/README.md

```diff
@@ -57,9 +57,11 @@ The checkpoint body is of the form:
 The first 3 lines of the body **MUST** be present in all Checkpoints.
 
 * `<Origin string>` should be a unique identifier for the log identity which issued the checkpoint.
-  The exact format of this is left undefined, but best practice is to have a scheme-less URI-like
-  prefix that will be globally unique, and give some idea of the log location/operator,
-  e.g. `example.com/log42`.
+  The exact format of this is left undefined, but best practice is to use a URI-like strucutre like
+  `<dns_name>[/<suffix>]`, where the log operator controls `<dns_name>`, e.g `example.com/log42`.
+  With a globally unique `<dns_name>`, this structure reduces the likelihood of origin collision,
```

Comment:
What are you trying to say by "globally unique" here? It could be read to say that the <dns_name> alone should be sufficient to identify a log, which is not at all correct. i.e. I believe `example.com/log42` and `example.com/log99` are both legit origins, even though they have the same <dns_name>

Reply:
👍